backend/src/services/v2/model.ts

```diff
@@ -116,13 +119,14 @@ export async function searchModels(
     }
   }
 
-  const results = ModelModel
+  const results = await ModelModel
     // Find only matching documents
```

Comment:
I don't know if these comments really add much, it seems quite self explanatory to me